chat-commands.js

```diff
@@ -3676,8 +3679,8 @@ exports.commands = {
 		if (user.locked && !targetUser.locked) {
 			return this.popupReply(`You are locked and cannot challenge unlocked users.`);
 		}
-		if (user.battleBanned) {
-			return this.popupReply(`You are battle banned and cannot challenge users.`);
+		if (user.battlebanned) {
+			return this.popupReply(`You are banned from battling and cannot challenge users.`);
```

Comment:
I still think this should say the reason, if there was one

Reply:
Nevermind, reasons are required - this should say it then

Reply:
I guess I should just do that for all of them